src/OpenLoco/src/Ui/ViewportInteraction.cpp

```diff
@@ -1210,7 +1210,7 @@ namespace OpenLoco::Ui::ViewportInteraction
                 {
                     if (track->owner() == CompanyManager::getControllingId())
                     {
-                        Ui::Windows::Construction::openAtTrack(window, track, interaction.pos);
+                        Ui::Windows::Construction::openAtTrack(*window, track, interaction.pos);
```

Comment:
do you not need to check for nullptr in all of these locations now?

Reply:
Inside `openAtTrack` the `Window` pointer wasn't already checked before changing it to reference. However, by looking at call hierarchy, I see `openAtTrack` is called only by `handleRightReleased` which in turn is called only by `stateViewPortRight` which checks for nullptr.